scripts/start_gaudi_vllm_server.sh

```diff
@@ -119,6 +119,7 @@ fi
 
 if [ "$num_hpu" -gt 1 ]; then
     export PT_HPU_ENABLE_LAZY_COLLECTIVES=true
+    export VLLM_EP_SIZE=$num_hpu #TODO: remove the export when --enable-expert-parallel is enabled.
```

Comment:
VLLM_EP_SIZE already set at https://github.com/HabanaAI/vllm-fork/blob/7017e1da6b50dc4e3e9a7a0a0777cdf086bd559c/scripts/utils.sh#L22

Reply:
the code is removed.